cloud-init-configs/auto-ssh.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+USER="$1"
```

Comment:
Why are you using variables here?

Reply:
host machine has different username than vm's and im not sure how else I would get the VMNAME when I'm running multiple vm's

Reply:
I see what you're saying now. I thought you didn't want any variables what-so-ever.